docs/core/install/includes/linux-install-80-apk.md

```diff
@@ -0,0 +1,28 @@
+---
+author: adegeo
+ms.author: adegeo
+ms.date: 11/14/2023
+ms.topic: include
+---
+
+### Install the SDK
+
+The .NET SDK allows you to develop apps with .NET. If you install the .NET SDK, you don't need to install the corresponding runtime. To install the .NET SDK, run the following command:
+
+```bash
+sudo apk add dotnet8-sdk
+```
+
+### Install the runtime
+
+The ASP.NET Core Runtime allows you to run apps that were made with .NET that didn't provide the runtime. The following command installs the ASP.NET Core Runtime, which is the most compatible runtime for .NET. In your terminal, run the following command:
```

Comment:
I present it more like this is *the* runtime you want to install, not *only* if you want web-based apps. After this has commands to install **just** the non-ASP.NET Core runtime if that's what you want. Otherwise, we always recommend the ASP.NET Core runtime regardless.

Reply:
Gotcha - makes sense.